plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -28,7 +28,25 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-NEVRA = namedtuple('NEVRA', ['name', 'epoch', 'version', 'release', 'arch'])
+_NEVRA = namedtuple('NEVRA', ['name', 'epoch', 'version', 'release', 'arch'])
+
+
+class NEVRA(_NEVRA):
+    @classmethod
+    def _fromdict(cls, nevra_dict):
+        """Given a dict with NEVRA keys, return a NEVRA namedtuple using its values
+
+        :param nevra_dict: dict with NEVRA keys (name, epoch, version, release, arch)
```

Comment:
Oh wait, I'm silly. It's NEVRA.__new__ that explodes, and I made _fromdict to explicitly do this. I'll just update the docstring. :D